kong/clustering.lua

```diff
@@ -459,7 +481,8 @@ function _M.handle_cp_websocket()
             ngx_log(ngx_DEBUG, "sent PONG packet to data plane")
           end
 
-        else -- config update
+        elseif should_send_config_update(KONG_VERSION, node_version) then
+          -- config update
```

Comment:
We should make this communication significantly more mature.
Can the control plane send the reason for not sending configuration to the data plane?

Reply:
Discussed with Harry and we will do this after the 2.3 alpha but before final in an interest of time here.